gpii/node_modules/eventLog/src/eventLog.js

```diff
@@ -78,10 +92,22 @@ fluid.defaults("gpii.eventLog", {
             args: ["{that}", "stop"]
         }
     },
-
-    logDestination: "tcp://127.0.0.1:51481"
+    // File path, or tcp://host:port
+    logDestination: null
```

Comment:
Question, does this replace logPath at https://github.com/GPII/universal/pull/790/files#diff-daf31e2b6c3b18e344aa3ded8a5ae009R69 ? If so, then there is some places where needs to be updated.

Reply:
Bump

Reply:
No. `logPath` (and `logServer`) members are filled in using `options.logDestination`, depending on whether it's a local path or a tcp:// uri. Added comments for this.

`options.logDestination` gets set from config.